mappings/net/minecraft/world/gen/GeneratorOptions.mapping

```diff
@@ -38,9 +38,17 @@ CLASS net/minecraft/class_5285 net/minecraft/world/gen/GeneratorOptions
 		ARG 0 biomeRegistry
 		ARG 1 chunkGeneratorSettingsRegistry
 		ARG 2 seed
+	METHOD method_28608 getRegistryWithReplacedOverworldGenerator (Lnet/minecraft/class_2378;Lnet/minecraft/class_2370;Lnet/minecraft/class_2794;)Lnet/minecraft/class_2370;
+		ARG 0 dimensionTypeRegistry
+		ARG 1 optionsRegistry
+		ARG 2 generator
 	METHOD method_28609 getDimensions ()Lnet/minecraft/class_2370;
 	METHOD method_28610 validate ()Lcom/mojang/serialization/DataResult;
 	METHOD method_28611 isStable ()Z
 	METHOD method_29575 getWorlds ()Lcom/google/common/collect/ImmutableSet;
+	METHOD method_29962 getRegistryWithReplacedOverworld (Lnet/minecraft/class_2370;Ljava/util/function/Supplier;Lnet/minecraft/class_2794;)Lnet/minecraft/class_2370;
```

Comment:
I'm not so sure about this name, it's very long. Should I change it to something like `withReplacedOverworld`, or should I keep it like this?

Reply:
I think this name is good enough. Someone else is free to suggest more concise names.